scripts/install_platform.sh

```diff
@@ -13,7 +13,6 @@ stage "Install ostis-web-platform"
 
 "${PLATFORM_PATH}/scripts/install_submodules.sh"
 "${PLATFORM_PATH}/scripts/install_dependencies.sh" --dev
-"${PLATFORM_PATH}/scripts/build_sc_machine.sh"
-"${PLATFORM_PATH}/scripts/build_sc_web.sh"
+"${PLATFORM_PATH}/scripts/build_sc_machine.sh" --cm
```

Comment:
This script also cannot be changed, like all the others, they can be extended

Reply:
I propose to implement core installation and build logic in this scripts throught option passing (f. e. `--core`)